packages/docs/content/08-user-management.md

```diff
@@ -51,3 +37,17 @@ Spectro Cloud allows the users that have been added to be allowed or restricted
 The tenant admin can allow or restrict access of resources to users which can differ as per the scenario. A user can have complete access to a specific project but can be restricted access to other projects in which there is no involvement. An intermediate stage is also possible where read-only access can be provided in some projects. The <Tooltip trigger={<u>Roles</u>}>A <a href="/user-management/rbac#roles">Role</a> is a collection of permissions.</Tooltip> and <Tooltip trigger={<u>Permissions</u>}><a href="/introduction/concept-overviews#permission">Permissions</a> are associated with specific actions within the platform.</Tooltip> sections in the RBAC page provide more details on this.
 
 To add a user to a project, sign in as an admin and go to the `Users and Teams` section of the [Admin Dashboard](/getting-started#admindashboard). Click on the user that you want to enable access to. In the role editor that opens to the side, click `Add Role` in the "PROJECT ROLES" section. Select the required project from the drop-down menu and enable the roles as needed.
+
+# Multi-Organization Support for Users
+
+Palette is incorporating multi-organization support for its users. With this feature, we provide our users w$
```

Comment:
@resmaa seems like there is a truncation here. Please fix.

Reply:
@truegilb  thank you..resolved